book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
```

Comment:
- [ ] You must have forgotten to initialize @rentals, it is needed for the data to be pushed.